### PR TITLE
Updated for latest Nim version

### PR DESCRIPTION
**Pull request by andreaferretti**
Removed expr/stmt metatypes to make it compile + removed deprecations. Fixes https://github.com/oprypin/nim-random/issues/19

### CODE REVIEW

File: test/test.nim [hunk 1]
```diff
@@ -1,3 +1,3 @@
-import random.mersenne
-import random.xorshift
+import random/mersenne
+import random/xorshift
```
Comment (oprypin) [inline test/test.nim:2]: Is this change necessary?

**Review by oprypin — APPROVED** [review]

File: test/test.nim (hunk 1 above)
Comment (andreaferretti) [reply test/test.nim:2]: Yes, importing modules using dots has just been deprecated

**Comment by oprypin** [comment]
Thanks!
